src/Sylius/Bundle/ShopBundle/Resources/views/Account/AddressBook/_defaultAddressForm.html.twig

```diff
@@ -0,0 +1,8 @@
+{% set id = app.request.attributes.get('id') %}
+
+{{ form_start(form, {'action': path('sylius_shop_account_address_book_set_default', {'id': id}), 'method': 'PATCH'}) }}
+    <input hidden name="{{ form.defaultAddress.vars['full_name'] }}" value="{{ id }}" />
```

Comment:
We shouldn't add `name="{{ form.defaultAddress.vars['full_name'] }}"` just for Behat.